### PR TITLE
Contract response includes customer name [TP#935]

### PR DESCRIPTION
**Pull request by josephbridgwaterrowe**
https://westernmilling.tpondemand.com/entity/935

### CODE REVIEW

File: app/views/api/v1/commodity_merchandising/contracts/index.json.jbuilder [hunk 1]
```diff
@@ -2,6 +2,10 @@ json.array! contracts do |contract|
   json.balance contract.balance
   json.contract_id contract.ContractId
   json.customer_id contract.CustomerId
```
Comment (wopr42) [inline app/views/api/v1/commodity_merchandising/contracts/index.json.jbuilder]: Should we remove this since we're adding it under `customer` now?

**Comment by wopr42** [comment]
One question, otherwise LGTM 👍

**Comment by cracell** [comment]
Same question as John but LGTM 👍

**Comment by josephbridgwaterrowe** [comment]
I believe that would be safe, I'll just make sure its not used in Otto... which I'm pretty sure its not.

**Pull request merged**